arangod/Aql/AqlCall.h

```diff
@@ -217,6 +217,9 @@ struct AqlCall {
   }
 
   auto requestLessDataThan(AqlCall const& other) const noexcept -> bool;
+  
+  /// @brief reusable const empty call object
+  static AqlCall const emptyCall;
```

Comment:
I don't think this helps performancewise since `AqlCall` should be trivially copyable and therefore a default constructed instance should be similarly cheap. However, if you prefer it for readability I am fine with that.

Reply:
Ok, I reverted all of the `AqlCall::emptyCall` changes.
This also simplifies the changes in this PR a bit.